2.1/Dockerfile

```diff
@@ -18,16 +35,6 @@ RUN set -x \
 	&& gosu nobody true \
 	&& apt-get purge -y --auto-remove ca-certificates wget
 
-# solves warning: "jemalloc shared library could not be preloaded to speed up memory allocations"
-RUN apt-get update && apt-get install -y --no-install-recommends libjemalloc1 && rm -rf /var/lib/apt/lists/*
-
-# https://github.com/docker-library/cassandra/pull/98#issuecomment-280761137
-RUN { \
-		echo 'Package: openjdk-* ca-certificates-java'; \
-		echo 'Pin: release n=*-backports'; \
-		echo 'Pin-Priority: 990'; \
-	} > /etc/apt/preferences.d/java-backports
```

Comment:
Doesn't this one still need OpenJDK from backports since it still uses `jessie`?

Reply:
Nope :smile: [openjdk-7-jre-headless](https://packages.debian.org/jessie/openjdk-7-jre-headless) is not in backports.